lib/collection/certificate.js

```diff
@@ -135,6 +136,26 @@ _.assign(Certificate.prototype, /** @lends Certificate.prototype */ {
 
         // test the input url against allowed matches
         return this.matches.test(url);
+    },
+
+    /**
+     * Allows the serialization of a {@link Certificate}
+     *
+     * This is overridden, in order to ensure that certificate contents are not accidentally serialized,
+     * which can be a security risk.
+     */
+    toJSON: function () {
+        var obj = PropertyBase.toJSON(this);
```

Comment:
Makes sense. `_.unset` could be used to replace the check -> delete sequence here.